src/commonmark/en/content/user/data-visualizer.md

```diff
@@ -438,15 +467,17 @@ Assigned Categories is a composite dimension that represents associated category
 
 Another way of adding assigned categories is by accessing the **Add Assigned Categories** option from the `Data` dimension's context menu (not available for `Gauge`, `Year over year` or `Single value`).
 
-## Adding more axes
+## Adding more axes and changing how different series are displayed
```

Comment:
For consistency in the **List of available options**, maybe we can add a **Series tab** section with a reference to the the **Adding more axes** section. 
E.g. 

> **Series tab**
> _Options for adding more axes and changing how different series are displayed are set in this section. Please see a detailed description of how this works in the corresponding section below._

Reply:
Done.

Reply:
Fixed